### ISSUE TITLE
Is it Possible to Combine OpenPCDet Checkpoints with Custom Pretrained Models for Enhanced Detection Accuracy?

### ISSUE THREAD

**Issue by sinatayebati** [open-mmlab/OpenPCDet#1581]
Is there a way to use a checkpoint published by OpenPCDet along with a checkpoint from my own pretrained model as "--pretrained_model" to initiate finetuning of the weights from OpenPCDet using my own pretrained model for a few epochs?

I'm currently experimenting with pretrained checkpoints and finetuning using SOTA models on the OpenPCDet framework to enhance detection accuracies. However, I've encountered difficulties in replicating the reported results for Kitti by training the OpenPCDet models  from scratch (without loading pretrained model) with identical configurations. My average precision (AP) results for pedestrians are consistently about 5% lower than the reported results. Even when attempting to use the pretrained checkpoint from my own model and finetuning with OpenPCDet, the improvements in AP remain inferior to the reported SOTA model results. 

Given this situation, I'm curious whether it's feasible to simply load the published checkpoints along with my own pretrained checkpoint and finetune for a few epochs to align my base accuracy with the reported team results.
